game-core/src/test/java/games/strategy/debug/error/reporting/UserErrorReportTest.java

```diff
@@ -70,6 +35,6 @@ void gameVersionIsSet() {
             .build()
             .toErrorReport()
             .getGameVersion(),
-        is(not(emptyString())));
+        not(IsEmptyString.isEmptyString()));
```

Comment:
`IsEmptyString#isEmptyString()` is deprecated.  See https://github.com/triplea-game/triplea/pull/4160/commits/47dabbf225883373b9828c54dd85f238b2cbc0ae.